python/pyarrow/ipc.pxi

```diff
@@ -120,6 +120,8 @@ cdef class IpcReadOptions(_Weakrefable):
     ----------
     ensure_native_endian : bool, default True
         Whether to convert incoming data to platform-native endianness.
+    ensure_memory_alignment : bool, default True
```

Comment:
Is there a particular reason this defaults to true in Python but to false in C++? If not, then being consistent would make things less confusing for the user, IMHO.

Reply:
Good catch, we changed this from `true` to `false` in C++ last minute, missed that, will align shortly.